src/ck-ldb/LBManager.C

```diff
@@ -397,9 +397,10 @@ void _loadbalancerInit()
   _lb_args.statsOn() =
       !CmiGetArgFlagDesc(argv, "+LBOff", "Turn load balancer instrumentation off");
 
-  // turn instrumentation of communicatin off at startup
-  _lb_args.traceComm() = !CmiGetArgFlagDesc(
-      argv, "+LBCommOff", "Turn load balancer instrumentation of communication off");
+  // turn instrumentation of communicatin on at startup
```

Comment:
Might as well fix this typo, too.
```suggestion
  // turn instrumentation of communication on at startup
```